client/js/components/statement/fragmentList/DpFragmentList.vue

```diff
@@ -31,14 +31,14 @@
 
 <script>
 import AnimateById from '@DpJs/lib/shared/AnimateById'
+import DpStatementFragment from './Fragment'
 import { mapMutations } from 'vuex'
-import StatementFragment from './Fragment'
 
 export default {
   name: 'DpFragmentList',
 
   components: {
-    'dp-statement-fragment': StatementFragment
+    'dp-statement-fragment': DpStatementFragment
```

Comment:
can now be simplified into:

```suggestion
    DpStatementFragment
```

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2820/commits/3eb2e119e526abb83a5732cbd6339f8e55dd46d1